### PR TITLE
ISPN-16261 ProtoStream 5.0.7.Final

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-16261

### CODE REVIEW

**Review by ryanemerson — APPROVED** [review]

**Review by ryanemerson — CHANGES_REQUESTED** [review]
It looks we didn't remove all of the old proto.lock files:

```
Error:  Failed to execute goal org.infinispan.maven-plugins:proto-schema-compatibility-maven-plugin:1.0.6.Final:proto-schema-compatibility-check (default) on project infinispan-commons: Execution default of goal org.infinispan.maven-plugins:proto-schema-compatibility-maven-plugin:1.0.6.Final:proto-schema-compatibility-check failed: IPROTO000039: Incompatible schema changes:
Error:  IPROTO000040: Incompatible @ProtoTypeId in 'org.infinispan.persistence.commons.CounterState', from 'null' to '4000'
Error:  IPROTO000040: Incompatible @ProtoTypeId in 'org.infinispan.persistence.commons.Storage', from 'null' to '4003'
Error:  IPROTO000040: Incompatible @ProtoTypeId in 'org.infinispan.persistence.commons.CounterType', from 'null' to '4002'
```

**Comment by tristantarrant** [comment]
I did remove them all. Unfortunately it seems like the protolock type id thing is missing some items

File: commons/all/proto.lock [hunk 1]
```diff
@@ -1,14 +1,32 @@
 {
   "definitions" : [ {
-    "protopath" : "generated:/:user.commons.proto",
+    "protopath" : "generated:/:global.commons.proto",
     "def" : {
       "enums" : [ ],
```
[the remark this answers is not in the document]
Comment (tristantarrant) [reply commons/all/proto.lock:5]: You must have broken it :)

[the remark this answers is not in the document]
Comment (tristantarrant) [reply commons/all/proto.lock:5]: Jokes aside, it was also empty before. Checking

**Comment by ryanemerson** [comment]
Merging without Jenkins CI as GH action builds all passed. Thanks @tristantarrant

**Pull request merged**